src/main/webapp/js/instructorCourseEdit.js

```diff
@@ -197,8 +197,8 @@ function checkPrivilegesOfTutorForInstructor(instrNum) {
 
 function checkPrivilegesOfCustomForInstructor(instrNum) {
 	var numOfInstr = $("form[id^='formEditInstructor']").length;
-	if (instrNum <= numOfInstr && INSTRUCTOR_COURSE_EDIT_INSTRUCTOR_ACCESS_LEVEL_WHEN_LOADING_PAGE.length >= instrNum 
-			&& INSTRUCTOR_COURSE_EDIT_INSTRUCTOR_ACCESS_LEVEL_WHEN_LOADING_PAGE[instrNum-1] === "Custom") {
+	if (instrNum <= numOfInstr && instructorCourseEditInstructorAccessLevelWhenLoadingPage.length >= instrNum 
+			&& instructorCourseEditInstructorAccessLevelWhenLoadingPage[instrNum-1] === "Custom") {
```

Comment:
Can, but better to be consistent and use single quote, our style guide from last time till now always suggests ' over " but it seems it wasn't followed, we should follow it whenever we can.

Usually, it is meant to help with distinguishing between JavaScript and HTML, JavaScript single, so that when we use double ", we know it's for HTML etc.